src/foam/nanos/auth/User.js

```diff
@@ -180,6 +180,25 @@ foam.CLASS({
       updateMode: 'RO',
       section: 'personal'
     },
+   {
+      class: 'String',
+      name: 'jobTitle',
```

Comment:
Really don't like this here, but have no other place for it at the moment. 
We are working through payroll, and HR app, and would like to move a number of properties off of user: department, organization, ....   to an 'employee' object of some type which is associated with a business and a user or with an organization if we build that out.     Anyway just future thoughts for now.